### PR TITLE
Feature/vtl jackson

### PR DESCRIPTION
**Pull request by NicoLaval**

### CODE REVIEW

File: docs/en/usage.md [hunk 1]
```diff
@@ -0,0 +1,130 @@
...
+            .build();
+    response.status(Http.Status.INTERNAL_SERVER_ERROR_500).send(jsonErrorObject);
+
+    return null;
+}
+```
+
+## Dataset serialization
+
+Instead of dealing yourself with Dataset objects, defined into Trevas Model module, a dedicated module based on Jackson is avaiable into Trevas.
```
Comment (hadrienk) [inline docs/en/usage.md:79]: *available

File: docs/en/usage.md [hunk 2]
```diff
@@ -0,0 +1,130 @@
...
+```java
+ObjectMapper mapper = new ObjectMapper();
+mapper.registerModule(new TrevasModule());
+```
+
+Giving it to your webserver, `Dataset`, `Component`, `Bindings` will be serialized/deserialized automatically.
+
+### Bindings JSON model
+
+Scalar objects & datasets are enable into JSON bindings.
```
Comment (hadrienk) [inline docs/en/usage.md:110]: *enabled? 
What about: Scalars and dataset variables can automatically be deserialized by the Jackson module.

**Comment by hadrienk** [comment]
> SonarCloud Quality Gate failed.
> 
> [Bug](https://sonarcloud.io/project/issues?id=InseeFr_Trevas&pullRequest=33&resolved=false&types=BUG) [A](https://sonarcloud.io/project/issues?id=InseeFr_Trevas&pullRequest=33&resolved=false&types=BUG) [0 Bugs](https://sonarcloud.io/project/issues?id=InseeFr_Trevas&pullRequest=33&resolved=false&types=BUG)
> [Vulnerability](https://sonarcloud.io/project/issues?id=InseeFr_Trevas&pullRequest=33&resolved=false&types=VULNERABILITY) [A](https://sonarcloud.io/project/issues?id=InseeFr_Trevas&pullRequest=33&resolved=false&types=VULNERABILITY) [0 Vulnerabilities](https://sonarcloud.io/project/issues?id=InseeFr_Trevas&pullRequest=33&resolved=false&types=VULNERABILITY) (and [Security Hotspot](https://sonarcloud.io/project/issues?id=InseeFr_Trevas&pullRequest=33&resolved=false&types=SECURITY_HOTSPOT) [0 Security Hotspots](https://sonarcloud.io/project/issues?id=InseeFr_Trevas&pullRequest=33&resolved=false&types=SECURITY_HOTSPOT) to review)
> [Code Smell](https://sonarcloud.io/project/issues?id=InseeFr_Trevas&pullRequest=33&resolved=false&types=CODE_SMELL) [A](https://sonarcloud.io/project/issues?id=InseeFr_Trevas&pullRequest=33&resolved=false&types=CODE_SMELL) [22 Code Smells](https://sonarcloud.io/project/issues?id=InseeFr_Trevas&pullRequest=33&resolved=false&types=CODE_SMELL)
> 
> [](https://sonarcloud.io/component_measures?id=InseeFr_Trevas&pullRequest=33&metric=new_coverage&view=list) [46.2% Coverage](https://sonarcloud.io/component_measures?id=InseeFr_Trevas&pullRequest=33&metric=new_coverage&view=list)
> [](https://sonarcloud.io/component_measures?id=InseeFr_Trevas&pullRequest=33&metric=new_duplicated_lines_density&view=list) [0.0% Duplication](https://sonarcloud.io/component_measures?id=InseeFr_Trevas&pullRequest=33&metric=new_duplicated_lines_density&view=list)

The coverage is pretty low, are equals/hashCode taken into account the main metrics? This can often pollute the report.

File: docs/en/usage.md (hunk 2 above)
Comment (NicoLaval) [reply docs/en/usage.md:110]: Enabled yes.
Don't understand your "what about", I tried below to explain that scalar go in directly, and that our Jackson module take care of dataset with a defined shape. Not so clear maybe?

**Pull request merged**